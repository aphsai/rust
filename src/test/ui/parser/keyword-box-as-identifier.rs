fn main() {
    let box = "foo"; //~ error: expected identifier, found reserved keyword `box`
}
