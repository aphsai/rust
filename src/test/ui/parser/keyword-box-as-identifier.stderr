error: expected identifier, found reserved keyword `box`
  --> $DIR/keyword-box-as-identifier.rs:4:9
   |
LL |     let box = "foo";
   |         ^^^ expected identifier, found reserved keyword
   |
help: you can escape reserved keywords to use them as identifiers
   |
LL |     let r#box = "foo";
   |         ~~~~~

error: aborting due to previous error

